Can't run `make bin` with goreleaser 2.3.2 and `--skip-publish`

When using version 2.3.2 of `goreleaser` (see the version below), and running `make bin`, I was get the following error:

```
goreleaser --snapshot --skip-publish --clean
  ⨯ command failed                                   error=unknown flag: --skip-publish
make: *** [bin] Error 1
```
this can be fixed by changing `--skip-publish` to `--skip=publish` on the Makefile.

Didn't submit a PR because I don't know when this changed, but I thought you should know.


```
  ____       ____      _
 / ___| ___ |  _ \ ___| | ___  __ _ ___  ___ _ __
| |  _ / _ \| |_) / _ \ |/ _ \/ _` / __|/ _ \ '__|
| |_| | (_) |  _ <  __/ |  __/ (_| \__ \  __/ |
 \____|\___/|_| \_\___|_|\___|\__,_|___/\___|_|
goreleaser: Deliver Go Binaries as fast and easily as possible
https://goreleaser.com

GitVersion:    2.3.2
GitCommit:     e8c2ef77358b4b1e59b904322824ccbfa3487ab2
GitTreeState:  false
BuildDate:     2024-09-17T12:30:49Z
BuiltBy:       goreleaser
GoVersion:     go1.23.1
Compiler:      gc
ModuleSum:     h1:/fSs+KwmCXCXN4eeI27z/xp1+FwIS4rWisL6aTUeiGw=
Platform:      darwin/arm64
```

--- a/.github/workflows/ci.yml
+++ b/.github/workflows/ci.yml
@@ -24,7 +24,7 @@ jobs:
       run: go test -race -coverprofile=./coverage.txt -covermode=atomic ./...
 
     - name: Check goreleaser file
-      uses: goreleaser/goreleaser-action@v2
+      uses: goreleaser/goreleaser-action@v6.2.1
       with:
         version: latest
         args: check --soft

--- a/.github/workflows/ci.yml
+++ b/.github/workflows/ci.yml
@@ -27,7 +27,7 @@ jobs:
       uses: goreleaser/goreleaser-action@v6.2.1
       with:
         version: latest
-        args: "check --soft"
+        args: check
 
     #- name: Upload coverage
     #  uses: codecov/codecov-action@v2

--- a/.github/workflows/ci.yml
+++ b/.github/workflows/ci.yml
@@ -27,7 +27,7 @@ jobs:
       uses: goreleaser/goreleaser-action@v2
       with:
         version: latest
-        args: check
+        args: check --soft
 
     #- name: Upload coverage
     #  uses: codecov/codecov-action@v2

--- a/.github/workflows/ci.yml
+++ b/.github/workflows/ci.yml
@@ -27,7 +27,7 @@ jobs:
       uses: goreleaser/goreleaser-action@v6.2.1
       with:
         version: latest
-        args: check --soft
+        args: "check --soft"
 
     #- name: Upload coverage
     #  uses: codecov/codecov-action@v2

--- a/.goreleaser.yml
+++ b/.goreleaser.yml
@@ -1,3 +1,4 @@
+version: 2
 env:
   - GO111MODULE=on
 before:

--- a/.goreleaser.yml
+++ b/.goreleaser.yml
@@ -36,11 +36,11 @@ archives:
   - name_template: '{{ .ProjectName }}_{{- if eq .Os "Darwin" }}macos_{{- else }}{{- tolower .Os }}_{{end}}{{- if eq .Arch "amd64" }}x86_64{{- else if eq .Arch "386" }}i386{{- else }}{{ .Arch }}{{ end }}{{ if .Arm }}v{{ .Arm }}{{ end }}'
     format_overrides:
     - goos: windows
-      format: zip
+      formats: ['zip']
 checksum:
   name_template: 'checksums.txt'
 snapshot:
-  name_template: "{{ .Tag }}-next"
+  version_template: "{{ .Tag }}-next"
 changelog:
   sort: asc
   filters:
@@ -87,3 +87,4 @@ publishers:
       - packages
     dir: "{{ dir .ArtifactPath }}"
     cmd: curl -F package=@{{ .ArtifactName }} https://{{ .Env.FURY_TOKEN }}@push.fury.io/{{ .Env.FURY_ORG }}/
+  

--- a/Makefile
+++ b/Makefile
@@ -1,4 +1,4 @@
 
 bin:
-	goreleaser --snapshot --skip-publish --clean
+	goreleaser --snapshot --skip=publish --clean
 

